### ISSUE TITLE
getoer2go disconnect

### ISSUE THREAD

**Issue by tim-moody** [iiab/iiab-admin-console#455]
This has been seen several times.
```
 File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 183, in <module>
    main()
  File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 85, in main
    err_num, err_str, oer2go_catalog = get_oer2go_cat()
  File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 157, in get_oer2go_cat
    url_handle = urllib.request.urlopen(adm.CONST.oer2go_cat_url)
  File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 517, in open
    response = self._open(req, data)
  File "/usr/lib/python3.9/urllib/request.py", line 534, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 1375, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/usr/lib/python3.9/urllib/request.py", line 1350, in do_open
    r = h.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1347, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 276, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response
Getting the OER2Go/RACHEL Catalog Failed.

```

**Comment by tim-moody** [comment]
try rewriting urllib.request with requests and adding timeout, but it's mostly under RACHEL's control.

**Comment by tim-moody** [comment]
#466 added more try/except logic but still does not catch this exception. In fact it is made worse:

 File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 157, in get_oer2go_cat
    url_handle = urllib.request.urlopen(adm.CONST.oer2go_cat_url)
  File "/usr/lib/python3.9/urllib/request.py", line 214, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.9/urllib/request.py", line 517, in open
    response = self._open(req, data)
  File "/usr/lib/python3.9/urllib/request.py", line 534, in _open
    result = self._call_chain(self.handle_open, protocol, protocol +
  File "/usr/lib/python3.9/urllib/request.py", line 494, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.9/urllib/request.py", line 1375, in http_open
    return self.do_open(http.client.HTTPConnection, req)
  File "/usr/lib/python3.9/urllib/request.py", line 1350, in do_open
    r = h.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1347, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 276, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 184, in <module>
    main()
  File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 85, in main
    err_num, err_str, oer2go_catalog = get_oer2go_cat()
  File "/opt/admin/cmdsrv/scripts/get_oer2go_catalog", line 162, in get_oer2go_cat
    err_str = "GET-OER2GO-CAT ERROR - " + str(exc.reason) +'\n'
AttributeError: 'RemoteDisconnected' object has no attribute 'reason'
Getting the OER2Go/RACHEL Catalog Failed.

**Comment by tim-moody** [comment]
fixed in #466
